### PR TITLE
Explained the new property SizeFactor

### PR DESCRIPTION
**Pull request by petterroea**

### CODE REVIEW

File: Fuse/05 - Triggers and Animators.md [hunk 1]
```diff
@@ -172,7 +172,10 @@ You can scale an element uniformly along all axes by using the `Factor` property
 <Scale Factor="2" Duration="0.4"/>
 ```
 
-`Scale` can be used relative to something using the `RelativeTo` property. At the moment the only value for this is `SizeChange`, which Scales relative to the change in size of the element.
+`Scale` can be used relative to something using the `RelativeTo` property. The two choices are:
+
+* `SizeChange` - resizes relative to the change in size of the element specified by the `RelativeNode` property.
+* `SizeFactor` - resizes with a factor relative to another element, specified by `RelativeNode`. `1` would make it the same size as the `RelativeNode`, `0.5` would make it half the size, and so on.
```
Comment (sebbert) [inline Fuse/05 - Triggers and Animators.md]: Explicitly mention the `Factor` property, i.e. "A factor of `1` would make(...)"

File: Fuse/05 - Triggers and Animators.md [hunk 2]
```diff
@@ -192,7 +195,10 @@ When used in concert with @(LayoutAnimation), `Resize` allows you to animate the
 <Resize RelativeTo="SizeChange" Duration="0.5" Easing="CircularInOut" />
 ```
 
-The only value `Resize` has for `RelativeTo` at the moment is `SizeChange`, which resizes relative to the change in size of the element.
+`Resize` has two options for `RelativeTo` at the moment:
+
+* `SizeChange` - resizes relative to the change in size of the element specified by the `RelativeNode` property.
+* `SizeFactor` - resizes with a factor relative to another element, specified by `RelativeNode`. `1` would make it the same size as the `RelativeNode`, `0.5` would make it half the size, and so on.
```
Comment (sebbert) [inline Fuse/05 - Triggers and Animators.md]: Resize does not have the `Factor` property, mention it using `X` and `Y` instead.

**Comment by sebbert** [comment]
I just realized I have already documented this in #72, but this is definitely an improvement.

**Comment by kristianhasselknippe** [comment]
merged by hand, dunno why github doesn't recognize it.

**Comment by petterroea** [comment]
:confetti_ball:

**Pull request closed without merging**